packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -1085,12 +1085,10 @@ export const studioLocaleStrings = defineLocalesResources('studio', {
   'new-document.open-dialog-aria-label': 'Create new document',
   /** Title for "Create new document" dialog */
   'new-document.title': 'Create new document',
-  /** More detailed alert text letting user know they have an out-of-date version and should reload */
-  'package-version.new-package-available.description': 'Simply reload to use the new version.',
   /** Label for button that will make the browser reload when users' studio version is out-of-date */
-  'package-version.new-package-available.reload-button': 'Reload',
+  'package-version.new-package-available.reload-button': 'Push to reload',
```

Comment:
Should this be `Click` ? `Push to reload` sounds weird to me

Reply:
Knut suggested "Push" to be less user input interface biased (eg "click" is mouse/trackpad, "tap" is touch). No strong feelings on my side.